neo/Persistence/RocksDB/RocksDBStore.cs

```diff
@@ -0,0 +1,108 @@
+using Neo.IO.Caching;
+using Neo.IO.Data.RocksDB;
+using Neo.IO.Wrappers;
+using Neo.Ledger;
+using RocksDbSharp;
+using System;
+using System.Reflection;
+using System.Text;
+
+namespace Neo.Persistence.RocksDB
+{
+    public class RocksDBStore : Store, IDisposable
+    {
+        private readonly RocksDBCore db;
+
+        public RocksDBStore(string path)
+        {
+            db = RocksDBCore.Open(new Options { CreateIfMissing = true, FilePath = path });
+
+            if (db.TryGet(db.SYS_Version, RocksDBCore.ReadDefault, new byte[0], out var value) &&
+                Version.TryParse(Encoding.UTF8.GetString(value), out var version) && version >= Version.Parse("2.9.1"))
+                return;
+
+            using (var batch = new WriteBatch())
+            {
+                var options = new ReadOptions();
+                options.SetFillCache(false);
+
+                // Clean entries
+
+                using (var it = db.NewIterator(db.SYS_Version, options))
+                {
+                    for (it.SeekToFirst(); it.Valid(); it.Next())
+                    {
+                        batch.Delete(it.Key());
```

Comment:
Is there any way to empty the db directly?

Reply:
Removing the column families i think, this need to clean all the DB? i was thinking only in the version

Reply:
Done